src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -2203,7 +2203,7 @@ private bool SkipCommentMultiSegment()
 
             if (marker != JsonConstants.Asterisk)
             {
-                ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfValueNotFound, marker);
+                ThrowHelper.ThrowJsonReaderException(ref this, ExceptionResource.ExpectedStartOfValueNotFound, JsonConstants.Slash);
```

Comment:
FYI @WinCPP, I made this fix to match the behavior for skipping comments in the single segment case (from https://github.com/dotnet/corefx/pull/35934):
https://github.com/dotnet/corefx/blob/27dae83598c87e3cf4b139c8c981c13fe8e9a81e/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs#L2115

Reply:
yup. Will rebase for the other PR.